### ISSUE TITLE
Add merge conflict resolving functionality to mini.git.

### ISSUE THREAD

**Issue by 231tr0n** [echasnovski/mini.nvim#1402]
### Contributing guidelines

- [X] I have read [CONTRIBUTING.md](https://github.com/echasnovski/mini.nvim/blob/main/CONTRIBUTING.md)
- [X] I have read [CODE_OF_CONDUCT.md](https://github.com/echasnovski/mini.nvim/blob/main/CODE_OF_CONDUCT.md)

### Module(s)

mini.git

### Description

Hi @echasnovski! Do you think something like https://github.com/akinsho/git-conflict.nvim is suited to be part of mini.git module?
Basically this plugin offers functionality to resolve merge conflicts and highlight them in a buffer properly.

**Comment by echasnovski** [comment]
Thanks for the suggestion!

I thought about it and the conclusion was that it seems to be too complicated to implement concisely. Plus does not 100% fits [the 'mini.git' scope](https://github.com/echasnovski/mini.nvim/blob/7ebfab26d77a4b9b05aaae565907e7fa4b2ee154/doc/mini-git.txt#L25-L27). While there are other great alternatives:
-[jesseduffield/lazygit](https://github.com/jesseduffield/lazygit)
- ["Conflict" target](https://github.com/echasnovski/mini.nvim/blob/7ebfab26d77a4b9b05aaae565907e7fa4b2ee154/doc/mini-bracketed.txt#L244) in 'mini.bracketed' (with ["x" as default suffix](https://github.com/echasnovski/mini.nvim/blob/7ebfab26d77a4b9b05aaae565907e7fa4b2ee154/doc/mini-bracketed.txt#L40)) which allows easier navigation to conflicts in already opened buffers.

Closing as not planned.
